packages/plans-grid-next/src/components/features.tsx

```diff
@@ -145,10 +145,19 @@ const PlanFeatures2023GridFeatures: React.FC< {
 											setActiveTooltipId={ setActiveTooltipId }
 											id={ key }
 										>
-											{ currentFeature.getTitle( {
-												domainName: paidDomainName,
-												planSlug,
-											} ) }
+											<>
+												{ currentFeature.getTitle( {
+													domainName: paidDomainName,
+													planSlug,
+												} ) }
+												{ currentFeature?.getSubFeatureSlugs?.()?.length ? (
```

Comment:
I agree with the general reasoning, but these sub-features are existing features in this case. I am not sure why it would be better to have `getSubFeatureSlugs` return an array of slugs and then fetch the feature object from the feature list when we can directly return the feature object.

If we want to make it generalized, then I think that we would do something like this:
```
getSubFeatureSlugs: () => [
  translate( 'Unlimited products' ),
  translate( 'Bulk discounts' ),
  ...
]
```

For the purpose of the experiment (even though we have been careful not to leak any experiment or `trailMap...` variables into the pricing grid), maybe the current approach is acceptable? We can easily change it later.

Reply:
I also agree that from the data design point of view an array of slugs will be more consistent and match the principle of data normalization. 

However, I'm fine with going with this approach for this experiment. We can reiterate on it once we indeed need the concept of feature sublists.

One request though: could we update the function name into `getSubFeatureObjects()`? `getSubFeatureSlugs()` is now incorrect if we go this direction :)